notebooks/community/migration/UJ5 unified AutoML Vision Video Classification.ipynb

```diff
@@ -29,7 +29,7 @@
         "id": "title:migration,new"
```

Comment:
<p>I suggest that this title becomes:</p><p> </p><p>"Vertex AI AutoML Image Object Detection"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/123/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Done